### PR TITLE
FLEX-3524 ~ Updates configuration tests

### PR DESCRIPTION
**Pull request by rlemmers**

### CODE REVIEW

**Comment by rlemmers** [comment]
Updated the code for public lighting, reopening the pull request.

File: cucumber-tests-platform-publiclighting/src/test/java/com/alliander/osgp/cucumber/platform/publiclighting/glue/steps/mocks/RelayMapConverter.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+     *
+     * @param relayMapString
+     *            the index, address and relay type to convert.
+     * @return the created RelayMap.
+     */
+    private static RelayMap convertStringsToRelayMap(final String relayMapString) {
+        final String[] fields = relayMapString.split(",");
+        final RelayType relayType = RelayType.valueOf(fields[2]);
+
+        return new RelayMap(Integer.parseInt(fields[0]), Integer.parseInt(fields[1]), relayType, null);
```
Comment (MilanJansen) [inline cucumber-tests-platform-publiclighting/src/test/java/com/alliander/osgp/cucumber/platform/publiclighting/glue/steps/mocks/RelayMapConverter.java]: Why give relayType its own variable and rcIndex and rcIntern not? 
And is it safe to parse a String to an Int without a try / catch?

File: cucumber-tests-platform-publiclighting/src/test/resources/features/osgp-adapter-ws-core/ConfigurationManagement/GetConfiguration.feature [hunk 2]
```diff
@@ -15,7 +15,7 @@ Feature: CoreConfigurationManagement GetConfiguration
       | DcLights          | <DcLights>          |
       | DcMap             | <DcMap>             |
       | RcType            | <RcType>            |
```
Comment (MilanJansen) [inline cucumber-tests-platform-publiclighting/src/test/resources/features/osgp-adapter-ws-core/ConfigurationManagement/GetConfiguration.feature]: Why is RcType not removed in this feature?

File: cucumber-tests-platform-publiclighting/src/test/java/com/alliander/osgp/cucumber/platform/publiclighting/glue/steps/mocks/RelayMapConverter.java (hunk 1 above)
Comment (rlemmers) [inline cucumber-tests-platform-publiclighting/src/test/java/com/alliander/osgp/cucumber/platform/publiclighting/glue/steps/mocks/RelayMapConverter.java]: I removed the separate statement for relayType. We pass the integers for the index and internal address in our test code. If we accidentally use non-numbers, we will get a failing test, which is the intention.

File: cucumber-tests-platform-publiclighting/src/test/resources/features/osgp-adapter-ws-core/ConfigurationManagement/GetConfiguration.feature (hunk 2 above)
Comment (rlemmers) [inline cucumber-tests-platform-publiclighting/src/test/resources/features/osgp-adapter-ws-core/ConfigurationManagement/GetConfiguration.feature]: I've removed RcType.

**Comment by rlemmers** [comment]
retest this please

**Comment by rlemmers** [comment]
retest this please

**Pull request merged**